From a9654bd435060650e3f88ac30b688d71249cb2e8 Mon Sep 17 00:00:00 2001
From: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
Date: Wed, 29 Nov 2017 15:24:15 +0530
Subject: [PATCH 226/767] net: ethernet: Remove PHY_INTERFACE_MODE_SGMII_2500

Remove PHY_INTERFACE_MODE_SGMII_2500 as the interface mode
is not defined correctly. Correct way is PHY_INTERFACE_MODE_2500SGMII

Signed-off-by: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 include/linux/phy.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/phy.h b/include/linux/phy.h
index 07de9d1d9335..6cd09098427c 100644
--- a/include/linux/phy.h
+++ b/include/linux/phy.h
@@ -88,7 +88,6 @@ typedef enum {
 	PHY_INTERFACE_MODE_XAUI,
 	/* 10GBASE-KR, XFI, SFI - single lane 10G Serdes */
 	PHY_INTERFACE_MODE_10GKR,
-	PHY_INTERFACE_MODE_SGMII_2500,
 	PHY_INTERFACE_MODE_MAX,
 } phy_interface_t;
 
-- 
2.17.0

